source/documentation/partials/_IE801.md

```diff
@@ -445,8 +445,7 @@
   </td>
   <td>
    <span>
-    BC12
-   </span>
+    <a href="business-codelists.html">BC12</a>
```

Comment:
The closing tag for the span has been removed here accidentally and needs closing after this line